tac/agents/v1/base/reactions.py

```diff
@@ -53,40 +50,39 @@
 STARTING_MESSAGE_ID = 1
 STARTING_MESSAGE_TARGET = 0
 
-AgentMessage = Union[ByteMessage, CFP, Propose, Accept, Decline, OutContainer]
-
 
 class ControllerReactions(ControllerReactionInterface):
     """The ControllerReactions class defines the reactions of an agent towards the ControllerAgent."""
 
-    def __init__(self, crypto: Crypto, liveness: Liveness, game_instance: GameInstance, out_box: 'OutBox', agent_name: str) -> None:
+    def __init__(self, crypto: Crypto, liveness: Liveness, game_instance: GameInstance, mailbox: MailBox, agent_name: str) -> None:
         """
         Instantiate the ControllerReactions.
 
         :param crypto: the crypto module
         :param liveness: the liveness module
         :param game_instance: the game instance
-        :param out_box: the outbox of the agent
+        :param mailbox: the mailbox of the agent
         :param agent_name: the agent name
 
         :return: None
         """
         self.crypto = crypto
         self.liveness = liveness
         self.game_instance = game_instance
-        self.out_box = out_box
+        self.mailbox = mailbox
         self.agent_name = agent_name
 
-    def on_dialogue_error(self, dialogue_error_msg: DialogueErrorMessage) -> None:
+    def on_dialogue_error(self, dialogue_error_msg: Message) -> None:
         """
         Handle dialogue error event emitted by the controller.
 
         :param dialogue_error_msg: the dialogue error message
 
         :return: None
         """
+        assert dialogue_error_msg.protocol_id == "oef"
         logger.warning("[{}]: Received Dialogue error: answer_id={}, dialogue_id={}, origin={}"
-                       .format(self.agent_name, dialogue_error_msg.msg_id, dialogue_error_msg.dialogue_id, dialogue_error_msg.origin))
+                       .format(self.agent_name, dialogue_error_msg.get("id"), dialogue_error_msg.get("dialogue_id"), dialogue_error_msg.get("origin")))
```

Comment:
shouldn't be `origin`

Reply:
it is, right?
`dialogue_error_msg.get("origin")))`

Reply:
yeah I saw now! sorry.